website/content/api-docs/secret/kv/kv-v2.mdx

```diff
@@ -561,6 +561,60 @@ $ curl \
     https://127.0.0.1:8200/v1/secret/metadata/my-secret
 ```
 
+## Patch Metadata
+This endpoint patches an existing metadata entry of a secret at the specified
+location. The calling token must have an ACL policy granting the `patch`
+capability. Currently, only JSON merge patch is supported and must be specified
+using a `Content-Type` header value of `application/merge-patch+json`. It does
+not create a new version.
+
+| Method  | Path                     |
+| :------ | :----------------------- |
+| `PATCH` | `/secret/metadata/:path` |
+
+### Parameters
+
+- `max_versions` `(int: 0)` – The number of versions to keep per key. If not
```

Comment:
Currently I think the defaults are set to be `-1`. If we do decide to keep them at `-1`, we should update them here as well.

Reply:
The default of -1 above is mainly a way to determine whether or not to include the user-provided value. The secret engine itself will [default to 0](https://github.com/hashicorp/vault-plugin-secrets-kv/blob/vault-4290-patch-metadata/path_metadata.go#L252) when the field is not provided given that `GetOk` will result in the zero-value.

Reply:
Ahh that makes sense!  Thanks!